src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -17,42 +17,60 @@ export const projectOverviewSchema = {
     properties: {
```

Comment:
What does this schema describe?

```suggestion
    description: "A high-level overview of a project. It contains information such as project statistics, the name of the project, what members and what features it contains, etc.",
    properties: {
```

Reply:
Literally the same things as the project overview screen in the UI